README.md

```diff
@@ -1,12 +1,8 @@
-[![Netlify Status](https://api.netlify.com/api/v1/badges/9a24d489-6f78-40fa-91cd-4144d2580d0b/deploy-status)](https://app.netlify.com/sites/p2phackclub/deploys)
-[![Dependabot Status](https://api.dependabot.com/badges/status?host=github&repo=thatrobotdev/p2phackclub)](https://dependabot.com)
+
 <p align="center">
-  <a href="https://www.hackclub.com">
-    <img alt="P2P Hack Club Logo" src="static/logo.svg" width="60" />
-  </a>
 </p>
 <h1 align="center">
-  P2P Hack Club
+  P2P <a href="https://hackclub.com/">Hack Club!
```

Comment:
This <a> tag doesn't have a closing </a> to end the link. Maybe we should wrap the entire "P2P Hack Club" text and link it to https://p2phack.club!